keras_resnet/blocks/_1d.py

```diff
@@ -61,17 +87,17 @@ def basic_1d(filters, stage=0, block=0, kernel_size=3, numerical_name=False, str
 
     def f(x):
         y = keras.layers.ZeroPadding1D(padding=1, name="padding{}{}_branch2a".format(stage_char, block_char))(x)
-        y = keras.layers.Conv1D(filters, kernel_size, strides=stride, use_bias=False, name="res{}{}_branch2a".format(stage_char, block_char), **parameters)(y)
-        y = keras_resnet.layers.BatchNormalization(axis=axis, epsilon=1e-5, freeze=freeze_bn, name="bn{}{}_branch2a".format(stage_char, block_char))(y)
+        y = keras.layers.Conv1D(filters, kernel_size, strides=stride, name="res{}{}_branch2a".format(stage_char, block_char), **convolution_kwargs)(y)
+        y = keras.layers.BatchNormalization(axis=axis, epsilon=1e-5, name="bn{}{}_branch2a".format(stage_char, block_char), **batch_normalization_kwargs)(y)
```

Comment:
Shouldn't this have some `training=False` argument?

Reply:
Yep. `**batch_normalization_kwargs`. The logic is here:

https://github.com/broadinstitute/keras-resnet/pull/40/files#diff-45a7689737303b65e69187734a1fd48fR59